crates/burn-import/onnx-tests/tests/argmax/argmax.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+# used to generate model: onnx-tests/tests/concat/concat.onnx
```

Comment:
It's just a comment, but might as well point to the right file: onnx-tests/tests/argmax/argmax.onnx